### DISCUSSION TITLE
Language setting during install

### DISCUSSION THREAD

**Discussion by selwynorren** [elementary/installer#547] [Q&A] [answered]
Hi all, so this is not a bug in itself, but just a confusion from my side most probably, so I thought best to create discussion first,before I consider a bug report. I hope that is the correct action.

When booting and the installer starts, it asks for my language, keyboard and regional settings. Works very well. OS installs and then I need to reboot.

The moment my computer has booted normally, I am asked the same question again (Slightly different option can now be selected), of Language, keyboard and regional settings.

Was this not supposed to be set under the installation? I am confused why I had to do this twice?

**Comment by o-alquimista** [comment]
This is by design. Installation is performed OEM-style. The end-user then configures it to their liking.

**Reply by o-alquimista** [reply, marked as answer]
Actually, according to [this issue](https://github.com/elementary/initial-setup/issues/51), this is a planned feature.
